### DISCUSSION TITLE
Is it not possible to use Modifiers on the nav tag?

### DISCUSSION THREAD

**Discussion by Smilesharks** [statamic/cms#5382] [Troubleshooting]
I want to split a navigation menu into two sections, is it possible to limit the first chunk and offset the second?

`{{ nav:main_nav limit="4" }}
{{ title }}
{{ /nav:main_nav }}`

`{{ nav:main_nav offset="4" }}
{{ title }}
{{ /nav:main_nav }}`

Thanks!

**Comment by edalzell** [comment]
Don't believe those are valid parameters for the `nav` tag.

You could maybe use two tags and check the `count`?
